### PR TITLE
Socialite provider for Coursera

### PR DESCRIPTION
**Pull request by davetosh**

### CODE REVIEW

**Comment by davetosh** [comment]
Sorry, I seemed to have borked the pull request.

**Comment by DraperStudio** [comment]
Looked fine to me except the removal of `.styleci.yml`. What was the issue you had with your PR?

**Comment by davetosh** [comment]
For some reason it failed all the tests. Let me try again.

**Comment by DraperStudio** [comment]
Just download the StyleCI patch and do `git apply patch.txt`.
